src/repository/groupRepository.js

```diff
@@ -1,28 +1,29 @@
 import { Model } from '../db/models/index.js';
-import { ErrorHander } from '../utils/errorHander.js';
+import { InternalServerError } from '../exceptions/http/internalServer.js';
 
 export const groupRepository = {
-  createGroup: async data => {
-    const { group_name, desc } = data;
-    const newGroup = await Model.GroupDetail.create({
-      group_name,
-      desc,
-    });
+  createGroup: async (data, options = {}) => {
+    const newGroup = await Model.GroupDetail.create(data.body, options);
```

Comment:
directly do this 
const newGroup = await Model.GroupDetail.create(data, options);
and no need to handle internal server error here 
handle only this in services catch and throw internal server error from there